journal/202101.md

```diff
@@ -0,0 +1,376 @@
+- prevent [duplicate](https://github.com/decred/dcrd/pull/2563) pending connections to same IP address and port
+- miscellaneous updates to test code, logging, docs, dependencies, minor fixes, etc
+
+A total of 47 PRs from 10 contributors were [merged](https://github.com/decred/dcrd/pulls?q=is%3Apr+merged%3A2021-01-01..2021-01-31+sort%3Aupdated-asc), adding 12K and deleting 7K lines of code.
+
+In progress:
+
+- introduce [Age-Partitioned Bloom Filters](https://github.com/decred/dcrd/pull/2579) to significantly reduce memory used for tracking what data other peers are known to have
+
+**[dcrwallet](https://github.com/decred/dcrwallet)**
+
+- VSP client was [rewritten](https://github.com/decred/dcrwallet/pull/1956) to improve background processing and support recovery of fee payment information after seed restores { better translation to User-English is welcome, I didn't quite grasp the first design goal of 1956 }
+- narrow [rules](https://github.com/decred/dcrwallet/pull/1961) of what is considered a possible CoinJoin { I got what this change does but now where it is used }
+
+The above changes were also backported to v1.6 release.
+
+**[Decrediton](https://github.com/decred/decrediton)**
+
+- new page showing a list of [unspent](https://github.com/decred/decrediton/pull/3098) outputs (UTXOs)
+- show VSPs in [random](https://github.com/decred/decrediton/pull/3114) order
+- update [ticket status](https://github.com/decred/decrediton/pull/3121) on starts, restarts and seed restores
+- remove usage of [CSRF](https://github.com/decred/decrediton/pull/3133) tokens when communicating to Politeia, to prevent privacy leaks
+- [auto-detect](https://github.com/decred/decrediton/pull/3033) if the wallet used mixing in the past
+- UI hints for [RFP](https://github.com/decred/decrediton/pull/3118) proposals and submissions
+- updated translations
+- a few updates to Trezor support
+- more automated [UI tests](https://github.com/decred/decrediton/pull/2659)
+- ~24 bug fixes and 3 UI tweaks
+
+Most of the above changes were included in v1.6 release and the rest will land in v1.6.1 patch release.
+
+**[Politeia](https://github.com/decred/politeia)**
+
+- show basic [proposal rules](https://github.com/decred/politeiagui/pull/2290) before the Submit button
+- remove calls to [dcrdata](https://github.com/decred/politeia/pull/1360) from politeiavoter command-line tool
+- UI for setting up and verifying [TOTP](https://github.com/decred/politeiagui/pull/2127) codes for 2FA
+
+CMS:
+
+- improved proposal [billing](https://github.com/decred/politeiagui/pull/2275) summary
+- improved [code stats](https://github.com/decred/politeiagui/pull/2279) display
+- bug fixes
+
+{ tlog update would be nice }
+
+**[vspd](https://github.com/decred/vspd)**
+
+- make sending of ticket's [parent](https://github.com/decred/vspd/pull/218) transaction mandatory as it greatly reduces the chance of ticket broadcast errors
+- update dependencies
```

Comment:
Could mention that vspd version 1.0.0 is now tagged and all VSP operators should upgrade their mainnet deployment.

1.0.0 marks the beginning of a proper release process, where VSP operators should only ever run tagged releases in production (until now, vspd has still been in dev and operators have just been deploying whatever was latest master)

Reply:
I missed that, thanks! Added.